### DISCUSSION TITLE
Is there a way to export a single notebook to html using MyST?

### DISCUSSION THREAD

**Discussion by edoelas** [executablebooks/meta#953] [Q&A]
I would like to know if it is possible to export a single notebook to HTML in a similar way to the command `jupyter nbconvert`. I don't want css or any other fancy stuff (except for the cell outputs if it is needed), just plain, static HTML.

**Comment by riziles** [comment]
@edoelas , why not just use nbconvert?

**Reply by edoelas** [reply]
I prefer MyST markdown flavor, it has some interesting features.

**Comment by choldgraf** [comment]
I believe you can do this with the Jupyter Book cli:

https://jupyterbook.org/en/stable/structure/toc.html#build-an-article-from-a-single-file

**Reply by edoelas** [reply]
This generates a full book (with all its project structure) with just one page, but not a single HTML page as I want.

**Reply by choldgraf** [reply]
Hmm - I know that @rowanc1 was playing around with static HTML output in Myst-js. Maybe he has an idea on this one?

**Reply by rowanc1** [reply]
We are tracking the actual feature for this [here](https://github.com/executablebooks/mystjs/issues/188), I don't think we are quite there yet for what @edoelas wants!

**Comment by riziles** [comment]
I think you want to try this: 

<https://myst-parser.readthedocs.io/en/latest/docutils.html>

Have not tested it myself.

**Reply by chrisjsewell** [reply]
It would be https://myst-nb.readthedocs.io/en/latest/docutils.html
You can get only the "body" html, but you do have to fiddle a bit
